backend/YouTubeV2.Application/YTContext.cs

```diff
@@ -18,6 +19,52 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             base.OnModelCreating(modelBuilder);
             modelBuilder.ApplyConfigurationsFromAssembly(typeof(UserEntityConfiguration).Assembly);
+
+            var roles = new[]
+            {
+                new Role
+                {
+                    Id = "39cc2fe2-d00d-4f48-a49d-005d8e983c72",
+                    Name = Role.Simple,
+                    NormalizedName = Role.Simple.ToUpper(),
+                },
+                new Role
+                {
+                    Id = "63798117-72aa-4bc5-a1ef-4e771204d561",
+                    Name = Role.Creator,
+                    NormalizedName = Role.Creator.ToUpper(),
+                },
+                new Role
+                {
+                    Id = "b3a48a48-1a74-45da-a179-03b298bc53bc",
+                    Name = Role.Administrator,
+                    NormalizedName = Role.Administrator.ToUpper(),
+                }
+            };
+
+            modelBuilder.Entity<Role>().HasData(roles);          
+
+            var user = new User
```

Comment:
is id required to specify? Maybe change the data so that it is obvious that it is a simple user eg simpleuser@mail.com. i think if you are doing the seeding you can also add sample creator and administrator